### ISSUE TITLE
Regarding the issue of Threejs version

### ISSUE THREAD

**Issue by MrQiangHu** [repalash/threepipe#7]
May I ask if you will consider upgrading to the version of Threejs? Are you currently using version 153?

**Comment by repalash** [comment]
Yes, it will be done but slowly and few versions at a time while trying to maintain backward compatibility. But important bug fixes and some new features from the new versions might be merged early. Is there anything specific you would like to see?

**Comment by MrQiangHu** [comment]
Currently, I'm trying to use it to integrate other existing libraries, such as the postprocessing library. However, I've found that the rendering pipeline of threepipe is different from that of threejs, and directly using third-party libraries for threejs will result in errors. I would really appreciate it if you could write more about custom rendering pipelines and plugins, as this would make it easier to integrate third-party libraries for threejs

**Comment by repalash** [comment]
Hey so the main render pipeline in threepipe uses the default EffectComposer from threejs with 2 default passes - ExtendedRenderPass and ScreenPass, one for rendering and other for copying to the scene. Both of these can be extended by various plugins to add effects like material extensions, screen space effects. And more passes can be added to be pipeline if required.
In general there.js passes can be used as is in your code wherever required, but to use it with the main render pipeline, it needs to be registered in the render manager, which places the pass at the correct place and renders it when required. See [PipelinePassPlugin](https://threepipe.org/docs/classes/PipelinePassPlugin.html) for an abstract plugin that provides the boilerplate to create a plugin that registers a custom pass in the pipeline. Check [NormalBufferPlugin](https://threepipe.org/docs/classes/NormalBufferPlugin.html) for an example of that.

There are some more details here - https://threepipe.org/guide/render-pipeline.html#render-pipeline

For using third party three.js libraries, you can probably override the version in package.json if you get any type errors - 
```
  "overrides": {
    "three": "$three",
    "@types/three": "$@types/three"
  }
```

**Comment by OstenL** [comment]
The mousewheel zoom in OrbitControls does not work with Orthographic camera in threejs version 153. There is a different implementation of this in newer versions of threejs so it would be appreciated if this could be fixed.

**Comment by repalash** [comment]
Hey @OstenL I am in the progress of updating till r160. It will be done soon. 
The issue with orthographic camera zoom looks like a bug, there are several changes in r155 in OrbitControls, will fix that along with that.

**Comment by repalash** [comment]
@OstenL Orthographic camera zoom issue has been fixed in version 0.0.39
You can try it out here - https://threepipe.org/examples/?q=camera#camera-ortho-uiconfig/

**Comment by repalash** [comment]
three.js version is now being regularly updated. Closing this issue for now, you can open a new issue for anything specific
